desktop/src/main/java/bisq/desktop/primary/main/content/chat/channels/PublicTradeChannelSelection.java

```diff
@@ -309,8 +311,7 @@ protected ListCell<ChannelItem> getListCell() {
                 @Override
                 protected void updateItem(ChannelItem item, boolean empty) {
                     super.updateItem(item, empty);
-                    if (item != null && !empty && item.getChannel() instanceof PublicTradeChannel) {
-                        PublicTradeChannel publicTradeChannel = (PublicTradeChannel) item.getChannel();
+                    if (item != null && !empty && item.getChannel() instanceof PublicTradeChannel publicTradeChannel) {
```

Comment:
We want to support Java 11 for Android, so we avoided to use features not supported by Java 11.

Reply:
Understood, will revert all such changes.